webpack.config.dev.js

```diff
@@ -12,15 +12,12 @@ const insertHotReload = (host, port, entries) => (
   R.map(R.compose(R.flatten, v => [v].concat(hotEntry(host, port))), entries)
 );
 
-const devConfig = {
+Object.assign(config, {
   context: __dirname,
   output: {
     path: path.resolve('./static/bundles/'),
     filename: "[name].js"
   },
-  module: sharedConfig.module,
-  sassLoader: sharedConfig.sassLoader,
```

Comment:
the sassLoader? I refactored the webpack configuration a little while ago to add the commons chunk plugin and this was removed from the shared config then - I had just missed this line here.